hr-data-generator/src/test/java/uk/gov/gchq/palisade/example/hrdatagenerator/EmployeeTest.java

```diff
@@ -40,7 +40,7 @@ public void generateEmployee() {
     @Test
     public void generateData() {
         try {
-            CreateData.main(new String[]{".data", "50", "1"});
+            CreateData.main(new String[]{"/data", "50", "1"});
```

Comment:
Hmm, what happens when you don't have privelidge to write to "/" (corporate laptop gang)

Reply:
This seems like something that needs to have a general discussion.